symplyphysics/laws/dynamics/deformation/shear_stress_is_shear_modulus_times_strain.py

```diff
@@ -0,0 +1,50 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## When an object is under a shearing stress, the shear stress applied to the object is proportional
+## to the shear modulus of the object and the ratio between the displacement of one end of the object
+## in the direction of the applied (shear) force and the length of the object.
+
+# Law: tau = G * (dx / L)
+## tau - [shear stress](../../hydro/shear_stress_is_shear_force_over_area.py)
```

Comment:
Maybe move that law to `deformation` as well?

Reply:
Of course, it could be done later separately